felix/fv/infrastructure/felix.go

```diff
@@ -223,8 +225,9 @@ func RunFelix(infra DatastoreInfra, id int, options TopologyOptions) *Felix {
 		"-P", "FORWARD", "DROP")
 
 	return &Felix{
-		Container:      c,
-		startupDelayed: options.DelayFelixStart,
+		Container:       c,
+		startupDelayed:  options.DelayFelixStart,
```

Comment:
nit: would be nice to be consistent between whether we pass individual fields from `TopologyOptions` (like `startupDelayed`) or pass the whole struct

Reply:
`startupDelayed` is internally modified so we need to initialize it from the options so that we do not modify the options